packages/admin/admin/src/common/toolbar/breadcrumb/ToolbarBreadcrumbs.tsx

```diff
@@ -1,5 +1,5 @@
-import { ComponentsOverrides, Link, Typography as MuiTypography, TypographyTypeMap } from "@mui/material";
```

Comment:
FYI: The old `ToolbarBreadcrumbs` will be removed in a separate PR.

Reply:
@thomasdax98 intends to make the toolbar refactor a non-breaking change. Would this be possible?

Reply:
Not really, the old Breadcrumbs were intended to be passed into the Toolbar, now they are part of the Toolbar. 
Also the old Breadcrumbs were barely (if ever) used in applications.

Reply:
I abandoned the idea of a non-breaking new toolbar. I think it's not possible